File: Julia Version/PlottingEigenvalues.jl
#Hamed Baghal Ghaffari
#2021-10-20
#This code Plots the Eigenvalues of the finite Fourier Transformation.
function PlottingEigenvalues(c,m)
    #T=zeros(1,m)
    T=[0.0 for i=1:m]
    for i=1:div(m,2)
        T[2*i-1]=abs(eveneigenvalueofprolate(c,m,i));
        T[2*i]=abs(oddeigenvalueofprolate(c,m,i));
    end
    return T
end

using Plots
pyplot()

F=PlottingEigenvalues(1,20);
G=PlottingEigenvalues(2,20);

plot(F)
plot!(G)
#plot([T[i] for i=1:10])

File: Python Version/VS Code py Format/PlottingEigenvalues.py
#Hamed Baghal Ghaffari
#2021-10-17
#This code Plots the Eigenvalues of the finite Fourier Transformation.
from oddeigenvalueofprolate import oddeigenvalueofprolate
from eveneigenvalueofprolate import eveneigenvalueofprolate

def PlottingEigenvalues(c,m,n):
    T=[]
    for i in range(n):
        T.append(abs(eveneigenvalueofprolate(c,m,2*i)))
        T.append(abs(oddeigenvalueofprolate(c,m,2*i+1)))
    return T



#print(PlottingEigenvalues(1,40,5))